### ISSUE TITLE
Hit Finder alarm is always on

### ISSUE THREAD

**Issue by kimadagem** [brandonhellman/mturk-suite#188]
The alarm is activated even though there's nothing at the top of the screen saying it's on. I checked Options; it doesn't look any different so I tried clicking the Alarm button and the message appeared. I went back to the Finder page & closed the Alarm message (X in the upper right corner); that used to turn it off but it isn't doing that anymore. Has something changed there?

(BTW - I don't know if it matters but I use an up-to-date version of Chrome in Windows 7 64-bit.)

**Comment by brandonhellman** [comment]
Everything around alarms seems to be working fine on my end. None of the recent updates touched this part of Finder so this shouldn't be acting any differently than before. Are you able to consistently reproduce this? Thanks for the OS version, always good to have addition info.

**Comment by kimadagem** [comment]
I didn't have much time to deal with this yesterday so I just tried it and - Yes, it is still happening. The alarm went off as soon as I started the Finder, even with no message at the top. I went into Settings and deliberately turned it on, after which the message showed up. It cycled a couple of times, then I hit the X in the upper right corner of the message box; that has always turned it off in the past but it didn't this time.

I have to admit (in case you haven't noticed :) ) that I'm new to this and still trying to figure it out. But I checked the wiki page and it looks like I'm doing things right to turn the alarm on and off. Am I missing something?
